index.html

```diff
@@ -196,53 +174,57 @@ <h3 id="resource_wide_defaults">Resource-wide defaults</h3>
 
 
 <div class="req" id="bp_not_only_default">
-<p class="advisement">Specifications MUST NOT assume that a document-level default is sufficient.</p>
+<p class="advisement">Specifications MUST NOT assume that a document-level default is sufficient.  Even If a resource-wide setting is available, specify field-based metadata to override the default.</p>
```

Comment:
`Even If` => `Even if`

Reply:
I don't think the addition is clear? Perhaps:

> Even if a resource-wide value is available, it needs to be possible to specify metadata to override that default.

Reply:
Fixed.